Mathlib/Topology/Basic.lean

```diff
@@ -1559,7 +1559,7 @@ theorem ContinuousAt.eventually_mem {f : X → Y} {x : X} (hf : ContinuousAt f x
   hf hs
 
 /-- Deprecated, please use `not_mem_tsupport_iff_eventuallyEq` instead. -/
-@[deprecated] -- 15 January 2024
+@[deprecated (since := "2024-01-15")]
```

Comment:
Github can't apply suggestions on deleted lines, so reformatting:
```suggestion
@[deprecated not_mem_tsupport_iff_eventuallyEq (since := "2024-01-15")]
```